### PR TITLE
Autopark level 15

### PR DESCRIPTION
**Pull request by AlexandarSviridenko**
Autopark level 15 done.

### CODE REVIEW

File: src/main/java/com/Autopark/TechnicalSpecialist.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package com.Autopark;
 
 import com.Autopark.Auto.Color;
-import com.Autopark.Auto.VehicleType;
+//import com.Autopark.Auto.VehicleType;
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/TechnicalSpecialist.java]: remove if it's not needed

File: src/main/java/com/Autopark/infrastructure/orm/service/PostgreDataBaseService.java [hunk 2]
```diff
@@ -132,6 +157,26 @@ public <T> Optional<T> get(Long id, Class<T> clazz) {
         return Optional.empty();
     }
 
+    @SneakyThrows
+    public <T> Optional<T> getColumn(Long id, Class<T> clazz) {
+        checkTableAnnotation(clazz);
+
+        String sql = "SELECT * FROM " + clazz.getDeclaredAnnotation(Table.class).name() +
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/infrastructure/orm/service/PostgreDataBaseService.java]: separate to the variable of the class

How it's did for another SQL requests

File: src/main/java/com/Autopark/infrastructure/orm/service/PostgreDataBaseService.java [hunk 3]
```diff
@@ -357,6 +402,16 @@ private String getIdFieldName(Field... fields) {
...
     }
 
+    private String getColumnFieldName(Field... fields) {
+        for (Field field : fields) {
+            if (field.isAnnotationPresent(Column.class)) {
+                return field.getAnnotation(Column.class).name();
+            }
+        }
+
+        throw new IllegalStateException("Table hasn't got 'ID' field");
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/infrastructure/orm/service/PostgreDataBaseService.java]: please, create your own exception

File: src/main/java/com/Autopark/infrastructure/threads/configurators/ScheduleConfigurator.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+            thread.setDaemon(true);
+            thread.start();
+
+            return null;
+        }
+
+        return methodProxy.invokeSuper(object, args);
+    }
+
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/infrastructure/threads/configurators/ScheduleConfigurator.java]: please, do it in the similar style

use here @SneakyThrows also for previous

File: src/main/java/com/Autopark/repairAuto/MechanicService.java [hunk 5]
```diff
@@ -1,62 +1,55 @@
...
-import com.Autopark.infrastructure.core.annotations.InitMethod;
 
 import java.util.*;
 
-import static com.Autopark.parser.ParserBreakingsFromFile.*;
-
 public class MechanicService implements Fixer {
     public static final String[] details = {"Фильтр", "Втулка", "Вал", "Ось",
             "Свеча", "Масло", "ГРМ", "ШРУС"};
+    long i = 1;
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/repairAuto/MechanicService.java]: add modificator

File: src/main/java/com/Autopark/TechnicalSpecialist.java (hunk 1 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/TechnicalSpecialist.java]: fixed

File: src/main/java/com/Autopark/infrastructure/orm/service/PostgreDataBaseService.java (hunk 3 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/infrastructure/orm/service/PostgreDataBaseService.java]: fixed

File: src/main/java/com/Autopark/infrastructure/threads/configurators/ScheduleConfigurator.java (hunk 4 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/infrastructure/threads/configurators/ScheduleConfigurator.java]: fixed

File: src/main/java/com/Autopark/repairAuto/MechanicService.java (hunk 5 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/repairAuto/MechanicService.java]: fixed

File: src/main/java/com/Autopark/infrastructure/orm/service/PostgreDataBaseService.java (hunk 2 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/infrastructure/orm/service/PostgreDataBaseService.java]: fixed

**Comment by AlexeyKatomachin** [comment]
please, resolve conflict

**Comment by AlexandarSviridenko** [comment]
> please, resolve conflict

Done

**Pull request merged**